client/src/app/categories/categories-page/category-add/category-add.component.html

```diff
@@ -0,0 +1,49 @@
+<mat-drawer-container
+  class="category"
+  autosize
+  [ngClass]="open ? '' : 'display-none'"
```

Comment:
[class.display-none]="!open"

Reply:
But anyway you shouldn't hide the drawer this way. The drawer has API for closing